episodes/07-working-with-files-and-folders.Rmd

```diff
@@ -70,6 +74,43 @@ Instead of typing `cd firstdir`, let's try to type `cd f` and then press the Tab
 
 Pressing tab at any time within the shell will prompt it to attempt to auto-complete the line based on the files or sub-directories in the current directory. Where two or more files have the same characters, the auto-complete will only fill up to the first point of difference, after which we can add more characters, and try using tab again. We would encourage using this method throughout today to see how it behaves (as it saves loads of time and effort!).
 
+With `Desktop`, `Documents` and `Downloads` folders living in the same directory. Let's say we want to change directory into the `Downlaods` folder while using the `Tab` auto-complete help.
+
+```bash
+cd D
+```
+If we press `Tab` at this point we can't really auto-complete because there are three different folders that start with the letter `D`.
+If we double press `Tab` the shell provides us with the options that start with the letter `D`
+
+```output
+Desktop    Documents   
+Downloads
+```
+
+Let's try again 
+
+```bash
+cd Do
+```
+
+With double press on `Tab` we get
+```output
+Documents    Downloads  
+```
+
+Finally if we try the auto-complete method with enough unique set of characters that identify the word `Downloads` bash can auto complete the word:
+```bash
+cd Dow
+```
+
+`Tab`
+The shell will auto complete your command:
+```bash
+cd Downloads
+```
+
+This feature can be very helpful for quicker access to files and folders with known names 
```

Comment:
```suggestion
This feature can be very helpful for quicker access to files and folders with known names.  It can also prevent typos, which is especially helpful in files with long names.  
```